content/en/ingress.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Ingress
+status: Feedback Appreciated
+category: technology
+tags: ["fundamental", "", ""]
+---
+
+## What it is
+
+An Ingress is an abstraction that helps to manage the traffic from the outside into a [Kubernetes](/kubernetes/) cluster. 
+It consists of two elements, the ingress resource, and the ingress controller. 
+The ingress resource is just a Kubernetes manifest file that lives along with other manifest files and allows for the configuration of traffic routing. 
+The ingress controller is the web server technology that actually performs the routing of the traffic according to the configuration in the ingress resource.
+
+## Problem it addresses
+
+Traditionally, users access deployed production-grade applications using URLs, typically HTTPS, but whenever the traffic increases, it leads to the need for an increase in the number of replicas of the pod to handle it. The pod is the smallest unit of workload in Kubernetes. It is the object that contains the application. A load balancer is present between the application and the users to handle traffic for a specific service.
+Cloud Native web applications consist of multiple services, where one or more of those need to be accessible over the internet for the users to visit using their browser. 
+To make these services user accessible while using Kubernetes to run this application, we need to expose each service of the application to the outside world. 
+The most straightforward way would be to use a Load Balancer type of Kubernetes Service. 
+Creating such a Kubernetes Load Balancer Service results in a new component on the underlying infrastructure. This not only introduces new costs and management overhead but also, each newly created Load Balancer has its own external IP address to access it which increases the complexity because as a user we don't want to browse different URLs just to access an application.
```

Comment:
```suggestion
Creating such a Kubernetes Load Balancer Service results in a new component on the underlying infrastructure. 
This not only introduces new costs and management overhead but also, each newly created Load Balancer has its own external IP address. 
This leads to a bad user experience because as a user we don't want to browse different URLs just to access an application.
```

Adding some line breaks and rephrasing slightly.